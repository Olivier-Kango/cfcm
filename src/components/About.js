import React from 'react';

const About = () => (
  <div>
    Qui sommes-nous?
  </div>
);

export default About;
