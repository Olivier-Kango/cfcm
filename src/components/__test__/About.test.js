import { render, screen } from '@testing-library/react';
import About from '../About';

describe('Testing the About page', () => {
  test('should render', () => {
    render(<About />);
    const about = screen.getByText(/Qui sommes-nous/i);
    expect(about).toMatchSnapshot();
  });
});
