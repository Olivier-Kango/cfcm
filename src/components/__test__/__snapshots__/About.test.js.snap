// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the About page should render 1`] = `
<div>
  Qui sommes-nous?
</div>
`;
